datalad/distribution/install.py

```diff
@@ -725,6 +731,10 @@ def __call__(
                     else:
                         installed_items.append(rec_installed)
 
+                except Exception:
+                    # Error itself should already be logged.
+                    lgr.info("{0} skipped.".format(subds))
```

Comment:
I got that -- but you cannot guarantee that you know from this msg what happened to make it skip.  That is why it never hurts to add at least some detail about the exception.  I already had to troubleshoot many of such points where no clear reason was logged and then things were just skipped.